packages/auth/amplify_auth_cognito/lib/amplify_auth_cognito.dart

```diff
@@ -5,6 +5,17 @@
 library amplify_auth_cognito;
 
 export 'package:amplify_auth_cognito_dart/amplify_auth_cognito_dart.dart'
-    hide AmplifyAuthCognitoDart;
+    hide
+        AmplifyAuthCognitoDart,
+        // TODO(Jordan-Nelson): determine if the following models should be
```

Comment:
Note: I'm not sure all of these should be exported from the _dart package but I think we can evaluate that more closely at a later date. Left this as a todo for now.